### PR TITLE
fix: Optimized getting row offset

### PR DESCRIPTION
**Pull request by pengfeixx**
Both forward and reverse reads are read only once

Log: Fixed some known issues

### CODE REVIEW

**Comment by pengfeixx** [comment]
\rebuild

**Review by rb-union — COMMENTED** [review]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

File: logViewerService/logviewerservice.cpp [hunk 1]
```diff
@@ -205,23 +260,19 @@ QStringList LogViewerService::readLogLinesInRange(const QString &filePath, qint6
...
-        }
-
-        if (offsetLine < lineCount)
-            lineCount = offsetLine;
-
-        startLine = offsetLine - lineCount;
+    qint64 startPosition = 0;
+    QString token = QCryptographicHash::hash(filePath.toUtf8(), QCryptographicHash::Md5).toHex();
+    if (m_logLineIndex.contains(token)) {
+        startPosition = readFileAndReturnIndex(filePath, startLine, m_logLineIndex[token], bReverse);
```
Comment (starhcq) [inline logViewerService/logviewerservice.cpp:266]: filePath下日志被清除，m_logLineIndex缓存中对应文件的行偏移量数据是否也需要清理

**Pull request closed without merging**